packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,200 @@
+// @flow strict
+import { type Node, type Element, useRef, useState, useEffect, useId } from 'react';
+import Box from './Box.js';
+import TapArea, { type OnTapType } from './TapArea.js';
+import Popover from './Popover.js';
+import Text from './Text.js';
+import Link from './Link.js';
+import Icon from './Icon.js';
+import Flex from './Flex.js';
+import Tooltip from './Tooltip.js';
+import { type Indexable, CompositeZIndex, FixedZIndex } from './zIndex.js';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider.js';
+
+type Props = {|
+  /**
+   * Supply a short, descriptive label for screen-readers to describe the popover content. Used for [accessibility](https://gestalt.pinterest.systems/web/popover#ARIA-attributes) purposes.
+   */
+  accessibilityPopoverLabel: string,
+  /**
+   * Specifies the preferred position of the tooltip and popover relative to HelpButton. See [Popover's ideal direction variant](https://gestalt.pinterest.systems/web/popover#Ideal-direction) to learn more.
+   */
+  idealDirection?: 'up' | 'right' | 'down' | 'left',
+  /**
+   * If provided, displays a link at the bottom of the popover message.
+   * - `href` is the URL the hyperlink points to.
+   * - `text` is the displayed text for the link. See the [link variant](https://gestalt.pinterest.systems/web/helpbutton#With-a-link) for more details.
+   * - Optionally use `accessibilityLabel` to supply a short, descriptive label for screen-readers to replace link texts that don't provide sufficient context about the link component behavior. Texts like "Click Here", or "Read More" can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the link text. It populates `aria-label`. Screen readers read the `accessibilityLabel` prop, if present, instead of the link text. See [ Link's accessibility guidelines](https://gestalt.pinterest.systems/web/link#Accessibility) for more information.
+   * - Optionally provide an `onClick` callback, which is fired when the link is clicked (pressed and released) with a mouse or keyboard. See [OnLinkNavigationProvider](https://gestalt.pinterest.systems/web/utilities/onlinknavigationprovider) to learn more about link navigation.
+   */
+  link?: {|
+    accessibilityLabel?: string,
+    href: string,
+    onClick?: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+    text?: string,
```

Comment:
`text: string`, text should be required.

Reply:
According of previous conversations between I and @ponori this props is not required to text, text should has a default text and if the user wants, he changes...
This alignments has changed?

Reply:
Learn more is not an accessible text, if text is undefined we should require accessibilityLinkLabel. I dont think we save much by adding this default text. 

https://gestalt.pinterest.systems/web/pageheader#Complementary-items PageHeader's helperLink has no label.

Reply:
DONE